src/Microsoft.AspNetCore.Server.Kestrel/Internal/Networking/UvWriteReq.cs

```diff
@@ -85,18 +85,30 @@ public void Init(UvLoopHandle loop)
                 _callback = null;
                 _state = null;
                 Unpin(this);
-
-                var block = start.Block;
```

Comment:
We used to "unpin" the blocks (though that never really worked properly). When we removed block.Unpin(), I guess this vestigial code was missed.